### PR TITLE
OG-805: deployer: extract "applyConfig" from actual "deploy"

### PR DESCRIPTION
**Pull request by drortirosh**

### CODE REVIEW

File: packages/deployer/src/deployUtils.ts [hunk 1]
```diff
@@ -151,3 +160,99 @@ export async function printRelayInfo (hre: HardhatRuntimeEnvironment): Promise<v
   console.log(chalk.white('Relayer register:'))
   console.log(chalk.grey(`gsn relayer-register -m $SECRET --network ${network.url} --relayUrl https://${hre.hardhatArguments.network}.v3.opengsn.org/v3 --token ${stakingTokenAddress} --stake ${stakingTokenValue} --wrap`))
 }
+
+export async function getEnv (hre: HardhatRuntimeEnvironment): Promise<{ deployer: string, deployments: DeploymentsExtension, env: Environment }> {
```
Comment (forshtat) [inline packages/deployer/src/deployUtils.ts:164]: We have `getEnvironment` global function. Please rename to `getDeploymentEnvironment`.

File: packages/deployer/src/applyConfig.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { applyDeploymentConfig } from './deployUtils'
```
Comment (forshtat) [inline packages/deployer/src/applyConfig.ts:1]: Don't you want to export these scripts as CLI commands? Why nort?

File: packages/deployer/src/deployUtils.ts [hunk 3]
```diff
@@ -151,3 +160,99 @@ export async function printRelayInfo (hre: HardhatRuntimeEnvironment): Promise<v
...
+
+  return {
+    deployer,
+    deployments,
+    env
+  }
+}
+
+// check if token's minimum stake is correct. return null if no need to change.
+async function updateTokenStakeOrNull (hub: Contract, tokenAddr: string, configMinimumStake: string): Promise<{ token: string, minimumStake: string } | null> {
```
Comment (forshtat) [inline packages/deployer/src/deployUtils.ts:180]: This is not actor, this is getter. `getTokenMinimumStake`?

File: packages/deployer/src/applyConfig.ts (hunk 2 above)
Comment (drortirosh) [reply packages/deployer/src/applyConfig.ts:1]: this is an ethers/hardhat script, like deploy. 
it also depends on the "deployments" created by deploy under this folder.

**Review by forshtat — APPROVED** [review]

**Pull request merged**